demo/README.md

```diff
@@ -9,125 +9,13 @@ To run the completed project in this folder, you need the following:
 
 If you don't have a Microsoft account, you can [sign up for the Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program) to get a free Microsoft 365 subscription.
 
-## Register an application
+## Configure the sample for user authentication
 
-You can register an application using the Azure Active Directory admin center, or by using the [Microsoft Graph PowerShell SDK](https://docs.microsoft.com/graph/powershell/get-started).
+Instructions for user authentication configuration are located in [BASIC-CONFIG.md](BASIC-CONFIG.md).
 
-**NOTE:** If you downloaded this code from [https://developer.microsoft.com/graph/quick-start](https://developer.microsoft.com/graph/quick-start), an app registration has already been created for you. However, if you want to use the app-only portion of this sample, you will need to modify the app registration as specified in [Configure app-only auth (AAD admin center)](#configure-app-only-auth-aad-admin-center) or [Configure app-only auth (PowerShell)](#configure-app-only-auth-powershell).
+## Enable app-only authentication
```

Comment:
Is there somewhere to state `application` permissions to make this clearer?  A link to the documentation about delegated vs application permissions could be useful.
https://docs.microsoft.com/en-us/graph/auth/auth-concepts#delegated-and-application-permissions